tslint.json

```diff
@@ -10,11 +23,13 @@
             "spaces"
         ],
         "no-duplicate-variable": true,
-        "no-eval": true,
         "no-internal-module": false,
         "no-trailing-whitespace": true,
         "no-unsafe-finally": true,
         "no-var-keyword": true,
+        "no-unused-variable": true,
+        "no-unused-new": true,
+        "no-unused-expression": true,
```

Comment:
Please include [rules](https://github.com/Microsoft/PowerBI-visuals-ChicletSlicer/blob/master/tslint.json#L57) that have been recommended by Avi.